build/configs/artik053/grpc/defconfig

```diff
@@ -770,6 +775,7 @@ CONFIG_TLS_MPI_MAX_SIZE=512
 CONFIG_WIFI_MANAGER=y
```

Comment:
WIFI_MANAGER is already enabled. This PR is not matched with commit title.

Reply:
@sunghan-chang 
I changed the commit title as following your comment